doc/source/api-key-creation.rst

```diff
@@ -0,0 +1,37 @@
+.. _api-key-creation:
+
+====================
+API Key Creation
+====================
+
+API keys are used to authenticate requests made using the Conduce REST API.  API keys are created by the user and should be protected like passwords.  Anyone with an API key can make requests to Conduce and perform operations as the user who requested the key.
```

Comment:
s/who requested the key/who created the key

Reply:
Am I being too literal.  I am distinguishing the user who requests that a key be created from the system that actually creates it.  I don't want the user to get the impression that they must create an API key and POST it to Conduce.

Reply:
But I think I see your point.  What about "the user who owns the key?"

Reply:
sure